package.json

```
@@ -14,16 +14,16 @@
   "author": "Tidepool",
   "license": "BSD-2-Clause",
   "dependencies": {
-    "amoeba": "0.4.1",
+    "amoeba": "0.6.0",
     "aws-sdk": "2.1.23",
     "bunyan": "1.4.0",
     "crypto-js": "3.1.2-5",
-    "hakken": "0.1.11",
-    "lodash": "3.4.0",
-    "moment": "2.9.0",
+    "hakken": "^0.1.12",
```

```suggestion
    "hakken": "0.1.12",
```
Our best practice guidance is to match exact versions.